### PR TITLE
Iow 523

### PR DESCRIPTION
**Pull request by ssoper-usgs**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: src/main/java/gov/usgs/wma/waterdata/PreProcess.java [hunk 1]
```diff
@@ -28,26 +27,27 @@ public ResultObject apply(RequestObject request) {
 
 	@Transactional
 	protected ResultObject processRequest(RequestObject request) {
-		LOG.debug("json_data_id: {}", request.getId());
+		Long jsonDataId = request.getId();
```
Comment (dsteinich) [inline src/main/java/gov/usgs/wma/waterdata/PreProcess.java:30]: I'm torn between getting the jsonDataId & partitionNumber here vs just passing the request object to each method. With just the two parameters it's ok, but if more would be added the method signatures would continue to expand...

Comment (ssoper-usgs) [reply src/main/java/gov/usgs/wma/waterdata/PreProcess.java:30]: Yeah I was juggling this as well... I think from a visual perspective, the easier these just-after-the-handler classes are to look at, the better.  Maybe splitting out the request properties in the shared DAO method that actually launches the jdbc call would be the best place to split it out.

**Comment by ssoper-usgs** [comment]
@dsteinich RE: should we pass the partition number out through the result object - I don't think we'll need to.  I think it can be treated a lot like we treat the json_data_id (just `id` in the state machine) where we send it out from the raw load lambda, and from there it's passed from lambda to lambda as part of a ... `timeSeries` or `fieldVisit` json.

File: src/main/java/gov/usgs/wma/waterdata/PreProcess.java (hunk 1 above)
Comment (dsteinich) [reply src/main/java/gov/usgs/wma/waterdata/PreProcess.java:30]: maybe just put the property's get method in the debug and eliminate the two variable here too?

**Pull request merged**